### DISCUSSION TITLE
highlight selected topo faces and topo edges in plotter

### DISCUSSION THREAD

**Discussion by dkowollik** [ansys/pyprimemesh#878] [Q&A] [answered]
Hi, 

Is it possible to highlight topo faces and topo edges in the plotting window by providing ids of those? If so, has somebody example code for this?
Thanks!

**Comment by waltersma** [comment]
@dkowollik there is an issue open for being able to modify plotting options for pyprimemesh objects in the plotter that I think should achieve your goal (see #835).

Cc @AlejandroFernandezLuces

**Reply by AlejandroFernandezLuces** [reply, marked as answer]
Yes, exposing `plotting_options` from PyVista should allow more customization of the meshes, although selecting IDs and highlighting them might take me a bit to implement. 

To answer your question, it's not currently possible, but I'll work on it next week 🙂
